New section on moral anti-realism (#115)
destinygg/destinypositions: docs/philosophy.md

- The best way to maximize my own experience is to help maximize others' experience as well as my own. I believe that humans have a unique capability for reciprocity, and that given this universal goal, pooling the efforts of a community of people will exponentially increase progress towards it. I (and most other people) also lose quality of life when those around me do (e.g. seeing people homeless, having friends with poor mental health, having ill family members, etc).
- Therefore, the best rules in a society are rules that provide the best outcomes for the greatest number of people. In philosophical jargon terms, this would make me a [Rule Consequestialist](https://plato.stanford.edu/entries/consequentialism-rule/).

## Moral Anti-realism

> The first foundational belief listed above is that "There is no such thing as morality", for which the philosophical term is [Moral Anti-realism](https://plato.stanford.edu/entries/moral-anti-realism/). In general, I am not a fan of conversations about [meta-ethics](https://plato.stanford.edu/entries/metaethics/) because I believe they generally reduce to incommensurable systems that become impossible to have a debate about. Instead, it seems that people are won over by whether such systems are intuitively satisfactory to them. I recognise, however, that to most people moral anti-realism seems like a dubious system as it is common to believe in some sort of moral truth, so it warrants a little more explanation here.

Reviewer: ```suggestion
The first foundational belief listed above is that "There is no such thing as morality", for which the philosophical term is [Moral Anti-realism](https://plato.stanford.edu/entries/moral-anti-realism/). In general, I am not a fan of conversations about [meta-ethics](https://plato.stanford.edu/entries/metaethics/) because I believe they generally reduce to incommensurable systems that become impossible to have a debate about. Instead, it seems that people are won over by whether such systems are intuitively satisfactory to them. I recognize, however, that to most people moral anti-realism seems like a dubious system as it is common to believe in some sort of moral truth, so it warrants a little more explanation here.
```

I have a firm belief that all of reality, aside from some skepticism about the nature of ["the mind"](https://plato.stanford.edu/entries/dualism/), is entirely physical. In other words: if something is real, it can be found in the physical world, tested for, and measured. The problem is that morality doesn't appear to fit these criteria, and instead exists purely conceptually. 

> As such, it is my conclusion that some moral fact-of-the-matter does not exist, and that moral statements like "Stealing is wrong" are, in a vacuum, either meaningless or wrong. It is worth noting that different schools of moral-antirealists argue whether such statements should be considered either meaningless (in the case of Non-Cognitivism) or systematically false (in the case of Error Theory). I have no firm stance on either of these, hence my use of both.

Reviewer: ```suggestion
As such, it is my conclusion that some moral fact-of-the-matter does not exist, and that moral statements like "Stealing is wrong" are, in a vacuum, either meaningless or wrong. It is worth noting that different schools of moral anti-realists argue whether such statements should be considered either meaningless (in the case of Non-Cognitivism) or systematically false (in the case of Error Theory). I have no firm stance on either of these, hence my use of both.
```

> Just because I think there is no such thing as morality, however, does not imply that I think moral statements are useless. They provide an important shorthand for conversations about how I would like people to act and how I would like society to function. As mentioned in the summary, there is a school of thought related to this called [Fictionalism](https://plato.stanford.edu/entries/fictionalism/#ViaEliAnt), which roughly equates to my beliefs on this. Without getting into the weeds to much, it claims that as long as moral statements reach their proposed goal (i.e. to make people act a certain way, to create agreement about how society should function, etc.) then we can treat them as if they are [truth-apt](https://www.oxfordreference.com/view/10.1093/oi/authority.20110803105953845) statements.

Reviewer 2: ```suggestion
Just because I think there is no such thing as morality, however, does not imply that I think moral statements are useless. They provide an important shorthand for conversations about how I would like people to act and how I would like society to function. As mentioned in the summary, there is a school of thought related to this called [Fictionalism](https://plato.stanford.edu/entries/fictionalism/#ViaEliAnt), which roughly equates to my beliefs on this. Without getting into the weeds too much, it claims that as long as moral statements reach their proposed goal (i.e. to make people act a certain way, to create agreement about how society should function, etc.) then we can treat them as if they are [truth-apt](https://www.oxfordreference.com/view/10.1093/oi/authority.20110803105953845) statements.
```
